docs/quickstart.md

```diff
@@ -0,0 +1,176 @@
+# Initial Setup / Operations
+
+`SSO` is used to provide single-sign-on authentication and authorization for internal web
+applications, however, there are a few one-time prerequisites that need to be handled manually.
+
+**NOTE: all commands should be performed on your HOST OS.**
```

Comment:
I think this is another holdover from the rig docs?  Let's drop it!

Reply:
ditching